x/staking/simulation/common_test.go

```diff
@@ -0,0 +1,11 @@
+package simulation_test
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func init() {
```

Comment:
Why do we need these init calls? Isn't `sdk.PowerReduction` already set to this value?

Reply:
For staking, we want to test it in PowerReduction = 10^18, to check overflow cases. cc. @sunnya97

Reply:
I see. Makes sense!